EIPS/eip-3722.md

```diff
@@ -0,0 +1,164 @@
+---
+eip: 3722
+title: Poster
+author: Auryn Macmillan (@auryn-macmillan)
+discussions-to: (https://ethereum-magicians.org/t/eip-poster-a-ridiculously-simple-general-purpose-social-media-smart-contract/6751)
+status: Draft
+type: Standards Track
+category: ERC
+created: 2021-07-31
+---
+
+# [Poster](https://github.com/ETHPoster/contract)
+## Simple Summary
```

Comment:
We recently deprecated the "Simple Summary" in favor of a description field in the preamble. Please see the [EIP template ](https://github.com/ethereum/EIPs/blob/master/eip-template.md)for an example.